docs/integrations/Webpack.md

```diff
@@ -0,0 +1,192 @@
+# Integration with Webpack
+
+This example shows how you can integrate the MAGE Web SDK with [Webpack](https://webpack.github.io/). Explaining
+Webpack itself is out of scope of this documentation, so please read about Webpack before getting started.
+
+
+## Testing your build configuration
+
+You can make production and development builds using Webpack by running one of the following two commands.
+
+> Production build
+
+```shell
+webpack -p
+```
+
+> Development build
+
+```shell
+webpack -d
+```
+
+If no errors appear, you will know your configuration worked and your project was built into the configured
+build-folder.
+
+
+## Getting started
+
+### Application setup (server)
```

Comment:
I would refer to the MAGE doc instead here. I don't see the point of copy-pasting code samples.